<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionBank</class>
 <widget class="QWidget" name="QuestionBank">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>574</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>788</width>
    <height>574</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QuestionnaireApp - Question bank</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="questionBankTabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="domainsAndChaptersTab">
        <attribute name="title">
         <string>DOMAINS &amp; AND CHAPTERS</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="domainsAndChaptersTitleLabel">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Available domains &amp; chapters</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="searchDomainTitleLabel">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Search for domains</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="searchDomainInput">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableWidget" name="domainTableWidget">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>-1</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QTableWidget
{
	font: 15px &quot;Times New Roman&quot;;
}

QHeaderView::section 
{
    background-color: silver;
    border: none;
	font-family: Times new Roman;
	font-size: 16px;
}

QTableWidget::item:selected
{ 
background-color: rgb(36, 36, 36);
color: white;
}</string>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <property name="gridStyle">
                <enum>Qt::DashLine</enum>
               </property>
               <property name="sortingEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="addDomainTitleLabel">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Add new domain</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="addDomainInput">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="addDomainButton">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>12</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Add domain</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="chaptersForLabel">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Chapters</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableWidget" name="chapterTableWidget">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>-1</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QTableWidget
{
	font: 15px &quot;Times New Roman&quot;;
}

QHeaderView::section 
{
    background-color: silver;
    border: none;
	font-family: Times new Roman;
	font-size: 16px;
}

QTableWidget::item:selected
{ 
background-color: rgb(36, 36, 36);
color: white;
}</string>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <property name="gridStyle">
                <enum>Qt::DashLine</enum>
               </property>
               <property name="sortingEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="addChapterTitleLabel">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Add new chapter</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="addChapterInput">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="addChapterButton">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>12</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Add chapter</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="infoLabel">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>You can delete a domain or a chapter by pressing delete button on your keyboard.</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="questionsTab">
        <attribute name="title">
         <string>QUESTIONS</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="questionsTitleLabel">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>These are the questions associated with your account:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="infoLabel1">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Click on the + button below to add a new question.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="infoLabel2">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Double click on a question to modify or delete it.</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLabel" name="searchQuestionsTitleLabel">
                 <property name="font">
                  <font>
                   <family>Times New Roman</family>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Search for questions</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="searchQuestionInput">
                 <property name="font">
                  <font>
                   <family>Times New Roman</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="questionTableWidget">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>-1</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QTableWidget
{
	font: 15px &quot;Times New Roman&quot;;
}

QHeaderView::section 
{
    background-color: silver;
    border: none;
	font-family: Times new Roman;
	font-size: 16px;
}

QTableWidget::item:selected
{ 
background-color: rgb(36, 36, 36);
color: white;
}</string>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustIgnored</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="gridStyle">
              <enum>Qt::DashLine</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="addQuestionButton">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>16</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
